src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql

```diff
@@ -49,6 +49,7 @@ WHERE
       "NCIT",
       "NDFRT",
       "NIFSTD",
+      "NORD",
```

Comment:
Hmm, I see these prefixes are supposedly illegal, but not sure why.

Reply:
@joeflack4 I just created these.

Reply:
These are the legal ones

Reply:
Ah I see now the `    FILTER( STRBEFORE(str(?source),":") not in (`